docs/team/joel.md

```diff
@@ -1,6 +1,55 @@
-# John Doe - Project Portfolio Page
+# Joel Ng Yi Xian - Project Portfolio Page
```

Comment:
Hi same as joseph you need to change the file name to your github username and update the aboutus